docs/content/docs/how-to/update-handwritten-documentation.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Update handwritten provider documentation"
+summary: "Handwritten resources and datasources have handwritten documentation that needs to be updated in PRs."
+weight: 24
+---
+
+# Update handwritten provider documentation (for handwritten resource or datasource)
+
+{{< hint info >}}
+**Note:** If you want to find information about documentation for a generated resource, look at the [Documentation](#documentation) page instead. The information on this page will not be relevant for resources that have generated documentation.
+
+{{< /hint >}}
+
+## How providers documentation works
+
+For general information about how provider documentation works, see [Documentation](#documentation).
```

Comment:
Ditto other comment about replacing link `[Documentation](#documentation)` with link to the general provider docs page